dev.sh

```diff
@@ -15,7 +15,7 @@ _start_gunicorn() {
   if [[ "$STATSD_HOST" ]]; then
     suffix="--statsd-host ${STATSD_HOST}:${STATSD_PORT}"
   fi
-  if [[ "$RUN_ENV" == "ENTERPRISE" ]] || [[ "$RUN_ENV" == "enterprise" ]] || [[ "$RUN_ENV" == "DEV" ]]; then
+  if [ "$RUN_ENV" == "ENTERPRISE" ] || [ "$RUN_ENV" == "DEV" ]; then
```

Comment:
Did you verify this works? Doing docker-compose up and ensuring the api pod runs the migrations script on its own would do it

Reply:
I did run the docker compose and verified that the api was running / I was able to access it. So... if that would do it I'd say I verified that it works, yes.